Ficek_QuantiApp/Ficek_QuantiApp/Domain/Sources/ErrorForAlerts/ErrorAlertState.swift

```diff
@@ -0,0 +1,46 @@
+import Foundation
+import ComposableArchitecture
+import SwiftUI
+import ErrorReporting
+
+public extension AlertState {
```

Comment:
Jinak tohle urcite neni target v `Domain` vrstve, ma to bejt ve `Feature` vrstve. Idealne treba `TCAExtensions` target, nebo `ErrorAlerts` nebo tak. Ale je to UI vec, nemuze to bejt v `Domain`.